classes/sourceconstructor.js

```diff
@@ -50,7 +50,11 @@ class SourceConstructor {
 		for (const [key, value] of Object.entries(sourceParams)) {
 			requestURL += key + "=" + value + '&';
 		}
+		if(this.apiName == "Eventbrite"){
```

Comment:
@ssgrover should this method be overwritten in the eventbrite.js file to include this chunk of code? That might not be as clean as this tho. Not sure

Reply:
Yes sourceconstructor shouldn't have any mention of other APIs. Better to move this to an override in eventbrite.js